package.json

```diff
@@ -44,12 +45,18 @@
     "mocha": "^1.21.4",
     "node-static": "^0.7.4",
     "object-assign": "^3.0.0",
+    "pre-commit": "^1.2.2",
     "replace-ext": "0.0.1",
     "request": "^2.44.0",
     "rimraf": "^2.5.4",
     "run-sequence": "^0.3.6",
     "sinon": "^1.10.3",
     "through2": "^2.0.0",
     "vinyl-sourcemaps-apply": "^0.1.4"
-  }
+  },
+  "pre-commit": [
+    "build",
+    "test",
+    "build:add"
```

Comment:
Sometimes this takes a while do run locally so we could add a check in travis.yml to verify that no files have been added/changed after running these files on CI.

Reply:
Oh, true, we have checks from Travis as well! I've been using the pre-commit test hook for the last week locally on my branch and haven't minded the test wait. But if it gets too slow with more tests (or feels prohibitively slow for others), yep, you're right, we can defer to Travis with that file check.